BackendApartmentReservation/BackendApartmentReservation/Repositories/EmployeeRepository.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using BackendApartmentReservation.Database;
+using BackendApartmentReservation.Database.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace BackendApartmentReservation.Repositories
+{
+    public class EmployeeRepository : IEmployeeRepository
+    {
+        private readonly DatabaseContext _context;
+
+        public EmployeeRepository(DatabaseContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<IEnumerable<DbEmployee>> GetAllEmployees()
+        {
+            return await _context.Employees.ToListAsync();
+        }
+
+        public async Task<DbEmployee> GetById(int employeeID)
+        {
+            return await _context.Employees.FindAsync(employeeID);
+        }
+
+        public async Task CreateEmployee(DbEmployee dbEmployee)
+        {
+            if (dbEmployee.FirstName != null && dbEmployee.LastName != null 
```

Comment:
Either throw `ArgumentNull` exception if some of the checks fail, or don't use them at all, since database will take care of that